src/components/LicenseRevokeModal/index.jsx

```diff
@@ -0,0 +1,191 @@
+
+    if (firstFocusableElement) {
+      firstFocusableElement.focus();
+    }
+  }
+
+  componentDidUpdate(prevProps) {
+    const {
+      submitFailed,
+      submitSucceeded,
+      onClose,
+      error,
+    } = this.props;
+
+    const errorMessageRef = this.errorMessageRef && this.errorMessageRef.current;
+
+    if (submitSucceeded && submitSucceeded !== prevProps.submitSucceeded) {
+      onClose();
+    }
+
+    if (submitFailed && error !== prevProps.error && errorMessageRef) {
+      // When there is an new error, focus on the error message status alert
+      errorMessageRef.focus();
+    }
+  }
+
+  handleModalSubmit() {
+    const {
+      licenseId,
+      user,
+      sendLicenseRevoke,
+    } = this.props;
+
+    const options = {};
+    options.assignments = [{
+      uuid: user.uuid, email: user.emailAddress, licenseStatus: user.licenseStatus,
+    }];
+
+    return sendLicenseRevoke(licenseId, options)
+      .then((response) => {
+        this.props.onSuccess(response);
+      })
+      .catch((error) => {
+        throw new SubmissionError({
+          _error: [error.message],
+        });
+      });
+    /* eslint-enable no-underscore-dangle */
+  }
+
+  renderBody() {
+    const {
+      user,
+      submitFailed,
+    } = this.props;
+
+    return (
+      <React.Fragment>
+        {submitFailed && this.renderErrorMessage()}
+        <div className="license-details mb-4">
+          <React.Fragment>
+            <p className="message" >Doing so will remove access to the learning catalog and the learner portal
+              for {user.emailAddress}. You will need to add a license to re-enable their learning.
+            </p>
+          </React.Fragment>
+        </div>
+      </React.Fragment>
+    );
+  }
+
+  renderErrorMessage() {
+    const modeErrors = {
+      revoke: 'Unable to revoke license',
+    };
+    const { error } = this.props;
+
+    return (
+      <div
+        ref={this.errorMessageRef}
+        tabIndex="-1"
+      >
+        <StatusAlert
+          alertType="danger"
+          iconClassName="fa fa-times-circle"
+          title={modeErrors.revoke}
+          message={error.length > 1 ? (
+            <ul className="m-0 pl-4">
+              {error.map(message => <li key={message}>{message}</li>)}
+            </ul>
+          ) : (
+              error[0]
+            )}
+        />
+      </div>
+    );
+  }
+
+  renderTitle() {
+    return (
+      <React.Fragment>
+        <span>
+          <i className="fa fa-exclamation-circle" aria-hidden="true" />
+          <small> Are you sure you want to revoke access?</small>
+        </span>
+      </React.Fragment>
+    );
+  }
+
+  render() {
+    const {
+      onClose,
+      submitting,
+      handleSubmit,
+    } = this.props;
+
+    return (
+      <React.Fragment>
+        <Modal
+          ref={this.modalRef}
+          title={this.renderTitle()}
+          body={this.renderBody()}
+          buttons={[
+            <div className="col-6">
```

Comment:
Also, please remove this class and col-12 from L146 as close button and Revoke button are showing differently due to it.

Reply:
+1